html/semantics/forms/the-input-element/datetime.html

```diff
@@ -6,6 +6,13 @@
     <link rel="author" title="Fabrice Clari" href="mailto:f.clari@inno-group.com">
     <link rel="author" title="Dimitri Bocquet" href="mailto:Dimitri.Bocquet@mosquito-fp7.eu">
     <link rel="help" href="https://html.spec.whatwg.org/multipage/#the-input-element">
+    <link rel="help" href="https://html.spec.whatwg.org/multipage/#dom-input-type">
```

Comment:
@Ms2ger you added/edited the spec links in this test. Do you think we should keep them?